src/app/layout.tsx

```
@@ -94,6 +143,15 @@ export default function RootLayout({ children }: { children: React.ReactNode })
             <head>
                 <meta name="color-scheme" content="light" />
 
+                {/* JSON-LD structured data */}
+                <script
+                    type="application/ld+json"
+                    dangerouslySetInnerHTML={{ __html: JSON.stringify(jsonLd) }}
```

Biome (2.3.14)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)